tests/_utils-tests/model.js

```diff
@@ -12,6 +12,7 @@ import Element from '/ckeditor5/engine/model/element.js';
 import Text from '/ckeditor5/engine/model/text.js';
 import Range from '/ckeditor5/engine/model/range.js';
 import Position from '/ckeditor5/engine/model/position.js';
+import CKEditorError from '/ckeditor5/utils/ckeditorerror.js';
```

Comment:
I remember that talked about this in the past. Do you recall whether there was some clear reason for not using `CKEditorError`? I think that it might be something that it simply doesn't make sense to use `CKEditorError` because it was meant to be minified in a special way.